testonly/hammer/hammer.go

```diff
@@ -454,10 +459,22 @@ func (w *writeWorker) writeOnce(ctx context.Context) (err error) {
 	if w.bias.invalid(ep, w.prng) {
 		glog.V(3).Infof("%d: perform invalid %s operation", w.mapID, ep)
 		invalidReqs.Inc(w.label, string(ep))
-		return w.s.setLeavesInvalid(ctx, w.prng)
+		return w.setLeavesInvalid(ctx, w.prng)
 	}
 	glog.V(3).Infof("%d: perform %s operation", w.mapID, ep)
-	return w.retryOp(ctx, w.s.setLeaves, string(ep))
+	return w.retryOp(ctx, w.setLeaves, string(ep))
+}
+
+func (w *writeWorker) nextKey() string {
+	w.keyIdx++
+	return fmt.Sprintf("key-%08d", w.keyIdx)
+}
+
+func (w *writeWorker) nextValue() []byte {
+	w.valueIdx++
+	result := make([]byte, w.s.cfg.LeafSize)
+	copy(result, fmt.Sprintf(valueFormat, w.valueIdx))
```

Comment:
Could you please inline the `valueFormat` here? It's only used once, and having it as a separate constant only harms readability. Also, it would be consistent with `nextKey()` function which does inline the format string.

Reply:
The reason this is still in the constants section is so that `minValueLen` can be defined alongside it. Separating them increases the probability of drift there. I'm not happy with this inconsistency here either, but I'll err on the side of reduced readability over probability of introducing a logic bug.